packages/core/src/settings.ts

```diff
@@ -69,18 +69,12 @@ export function loadEnvConfig(): Settings {
     // Node.js environment: load from .env file
     const envPath = findNearestEnvFile();
 
-    if (!envPath) {
-        throw new Error("No .env file found in current or parent directories.");
-    }
-
-    // Load the .env file
-    const result = config({ path: envPath });
+    // attempt to Load the .env file into process.env
+    const result = config(envPath ? { path: envPath } : {});
```

Comment:
Confirming that config({}) doesn't cause any errors?

Reply:
It does not, I checked